File: 3.cpp
#include <iostream>
using namespace std;
int main() {
    int amount = 0, number = 0;
    cin >> amount; // ввод количества чисел
    for (int i = 0; i < amount; i++) {
        cin >> number; // ввод всех чисел
        int count = 0;
        while (number != 0) {
            if ((number % 10) % 2 == 0) { // проверка на чётность каждой цифры
                count++; // подсчёт количества
            }
            number /= 10;
        }
        cout << "Result: " << count << endl;
    }
    return 0;
}

File: 3.go
package main

import (
    "fmt"
    )
func main() {
    var amount int
    fmt.Scan(&amount) // ввод количества чисел
    for i := 0; i < amount; i++ {
        var number int
        fmt.Scan(&number) // ввод чисел
        var count int = 0
        for number != 0 {
            if (number % 10) % 2 == 0 { // проверка на чётность цифр
                count++ // подсчёт количества
            }
            number /= 10
        }
        fmt.Println("Result: ", count)
    }
}